modules/observe/scorecards/con-understand-scorecard-plugins.adoc

```diff
@@ -0,0 +1,10 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="con-understand-scorecard-plugin_{context}"]
+= Understand Scorecard plugins in {product}
+
+include::{docdir}/artifacts/snip-developer-preview-scorecard.adoc[]
+
+The Scorecard plugin offers comprehensive, customizable insights into the health, security posture, and compliance of your components, services, and APIs. It centralizes Key Performance Indicators (KPIs) within {product} ({product-very-short}), so you do not have to consult multiple external systems.
```

Comment:
Can we also add how these plugins are beneficial for users? Current statements highlight what it does, but why a user would need it, that part is missing?